index.html

```diff
@@ -0,0 +1,630 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/032ed71b0e.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Responsive Design</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="first">
+            <ul class="first-nav">
+                <li><a href="#" class="white">News</a></li>
+                <li><a href="#" class="grey">Events <i class="fa fa-arrow-down smaller" ></i></a></li>
+                <li><a href="#" class="grey">Business <i class="fa fa-arrow-down smaller" ></i></a></li>
+                <li><a href="#" class="grey">AMAs</a></li>
+                <li><a href="#" class="grey">Space</a></li>
+                <li><a href="#" class="grey">Terms & Conditions </a></li>
+             </ul>
+                <ul class="left-nav">
+                <li><a href="#" class="grey">About</a></li>
+                <li><a href="#" class="grey">Advertise</a></li>
+                <li><a href="#" class="grey">Jobs</a></li>
+                <li><a href="#" class="grey">Contact</a></li>
+             </ul>
+             </div>          
+         <div class="get">
```

Comment:
You have a few problems here. Try to make this more as like the original. I can see that you improved from the last review and we are almost there. Just a few details more.

#### On mobile

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/87705097-c1835e80-c773-11ea-8701-538e3094a290.png) | ![image](https://user-images.githubusercontent.com/61479648/87705132-cfd17a80-c773-11ea-9195-ee5d6df2dfe2.png) |

- [ ] The search bar needs to be more close to the left end of the page. 
 

- [ ] There is no hamburger menu

- [ ] The navbar is occupying space from the first article. 

#### Large screen

- [ ] Pay attention to this TNW logo. It should be invisible on the large screen.

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/87705586-803f7e80-c774-11ea-8be2-dd54a83c2e77.png) | ![image](https://user-images.githubusercontent.com/61479648/87705667-a533f180-c774-11ea-88d1-94fc3121f2fd.png) |